prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1090,6 +1098,14 @@ int rxr_ep_init(struct rxr_ep *ep)
 
 		if (ret)
 			goto err_free;
+
+		ret = ofi_bufpool_grow(ep->rx_ooo_pkt_pool);
```

Comment:
If it does not already, `ofi_bufpool_create` should allow the caller to decide if they want lazy or immediate allocation.

Reply:
As far as I can tell, the function does not allow this option. Are you asking for a flag and check to be added in the bufpool create function to do so?

Reply:
Yeah, pretty much. This could be an attribute flag for the bufpool.

Reply:
@rajachan Such a change would require changing prototype of `ofi_bufpool_create`, which requires changing multiple provider (rxm, psm3). So I prefer to do that after 1.13.0 release.

Reply:
The attribute flags are opt-ins, like how we have the one for huge pages. I don't think you will need to modify providers if the default behavior is left at the lazy allocation; but post 1.13.0 sounds good to me.